config/default.js

```diff
@@ -0,0 +1,79 @@
+// CONFIG defaults (aka PRODUCTION)
```

Comment:
Yeah, I feel a better way around to have production config happen if you don't override something than to have development config if you don't override something.